### PR TITLE
Quadrat: Add new heading sizes and other small misc changes

### LINKED ISSUE #3640: Quadrat: Implement typography & color
Design here: https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/?node-id=50%3A766

Final color, typography and button styles: https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/Quadrat?node-id=760%3A112

- [x] Headings
- [x] Paragraphs
- [x] Blockquotes
- [x] Lists 
- [x] Tables
- [x] Code
- [x] Buttons

### PR DESCRIPTION
**Pull request by danieldudzic**
I have played around, trying to figure out the correct sizing for headings.

I have found it all a bit confusing because it seems that some of the Figma files contain an outdated font (Sora) and there seems to be a lot of font sizes being used:

- 12px
- 15px
- 17px
- 20px
- 22px
- 25px
- 30px
- 38px
- 40px
- 65px
- 70px
- 80px

### CODE REVIEW

File: quadrat/child-experimental-theme.json [hunk 1]
```diff
@@ -25,29 +25,34 @@
...
 				"fontSizes": [
+					{
+						"name": "Tiny",
+						"slug": "tiny",
+						"size": "15px"
+					},
 					{
 						"name": "Small",
-						"size": "17px",
+						"size": "22px",
```
Comment (jffng) [inline quadrat/child-experimental-theme.json]: These declarations generate presets available by default the editor. This makes me think they should follow the [paragraph sizing presented in the designs here](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/?node-id=443%3A4):

Screen Shot 2021-04-14 at 10 07 42 AM

For the heading sizes, I think we should make those custom variables that we can change based on a media query. Thoughts?

File: quadrat/style.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
 Theme Name: Quadrat 
-Theme URI: https://github.com/Automattic/themes/tree/master/mayland-blocks
```
Comment (jffng) [inline quadrat/style.css:3]: Thank you

File: quadrat/child-experimental-theme.json (hunk 1 above)
Comment (jffng) [reply quadrat/child-experimental-theme.json]: cc @beafialho

Comment (danieldudzic) [reply quadrat/child-experimental-theme.json]: +1 to the custom variables based on a media query for Headings.

Still not sure about the sizes for presets.

Comment (beafialho) [reply quadrat/child-experimental-theme.json]: Correct @jffng @danieldudzic, the final styles are on [this screen](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/Quadrat?node-id=760%3A112) in the file.

Comment (danieldudzic) [reply quadrat/child-experimental-theme.json]: Added a link to that screen in the main issue

**Review by MaggieCabrera — APPROVED** [review]
This LGTM!

**Pull request merged**